fbw-a380x/src/wasm/systems/a380_systems/src/pneumatic.rs

```diff
@@ -1429,31 +1429,43 @@ mod tests {
         thermodynamic_temperature::degree_celsius, velocity::knot,
     };
 
-    use crate::air_conditioning::A380PressurizationOverheadPanel;
+    use crate::air_conditioning::{A380AirConditioningSystem, A380PressurizationOverheadPanel};
 
     use super::{A380Pneumatic, A380PneumaticOverheadPanel};
 
     struct TestAirConditioning {
-        a380_air_conditioning_system: AirConditioningSystem<3, 2, 4>,
-
+        a380_air_conditioning_system: A380AirConditioningSystem,
         test_cabin: TestCabin,
+
         adirs: TestAdirs,
         pressurization: TestPressurization,
         pressurization_overhead: A380PressurizationOverheadPanel,
     }
     impl TestAirConditioning {
         fn new(context: &mut InitContext) -> Self {
-            let cabin_zones: [ZoneType; 3] =
-                [ZoneType::Cockpit, ZoneType::Cabin(1), ZoneType::Cabin(2)];
+            let cabin_zones: [ZoneType; 18] = [
+                ZoneType::Cockpit,
+                ZoneType::Cabin(11), // MAIN_DECK_1
+                ZoneType::Cabin(12), // MAIN_DECK_2
+                ZoneType::Cabin(13), // MAIN_DECK_3
+                ZoneType::Cabin(14), // MAIN_DECK_4
+                ZoneType::Cabin(15), // MAIN_DECK_5
+                ZoneType::Cabin(16), // MAIN_DECK_6
+                ZoneType::Cabin(17), // MAIN_DECK_7
+                ZoneType::Cabin(18), // MAIN_DECK_8
+                ZoneType::Cabin(21), // UPPER_DECK_1
+                ZoneType::Cabin(22), // UPPER_DECK_2
+                ZoneType::Cabin(23), // UPPER_DECK_3
+                ZoneType::Cabin(24), // UPPER_DECK_4
+                ZoneType::Cabin(25), // UPPER_DECK_5
+                ZoneType::Cabin(26), // UPPER_DECK_6
+                ZoneType::Cabin(27), // UPPER_DECK_7
+                ZoneType::Cargo(1),  // FWD
+                ZoneType::Cargo(2),  // BULK
```

Comment:
This looks a bit off vs what I have planned for the payload stations, but currently the related changes are in an outdated branch on the other repo.

Reply:
As discussed on Discord, this is fine, we'll find a way to "transfer" the number of passengers from the w/b stations to the air cond zones.